logstash-core/lib/logstash/pipeline.rb

```diff
@@ -176,8 +179,10 @@ def stopped?
   end
 
   def start_workers
-    @inflight_batches = {}
-
+    @filter_queue_client.set_events_metric(metric.namespace([:stats, :events]))
```

Comment:
Further to using the `set_*` idiom, using a 'setter' allows one to control the return value. For these kind of post initialize setup, I might decide to return `self` to enable setter chaining if I see a lot of post initialize setup in successive lines.